taskvine/src/manager/vine_manager.c

```diff
@@ -4250,28 +4276,10 @@ static double vine_task_priority(void *item)
 
 /* Put a given task on the ready list, taking into account the task priority and the manager schedule. */
 
-static void push_task_to_ready_list(struct vine_manager *q, struct vine_task *t)
+static void push_task_to_ready_tasks(struct vine_manager *q, struct vine_task *t)
 {
-	int by_priority = 1;
-
-	if (t->result == VINE_RESULT_RESOURCE_EXHAUSTION) {
```

Comment:
This is an important case that needs to be addressed.
An exhausted task should run prior to those of the same priority, so that we gain accurate resource info.